src/2_RTB/PianoDiProgetto/sections/AnalisiDeiRischi.typ

```diff
@@ -72,12 +72,22 @@ Si tratta della ridotta frequenza o della mancanza di contatti regolari e tempes
 
 === *RC4 - Mancanza di fiducia*
 I diversi livelli di competenze e una comunicazione poco trasparente possono portare allo sviluppo di un clima di sfiducia reciproca tra i membri del team: ciò potrebbe compromettere il successo del progetto.  
-  -*Identificazione*: la mancanza di fiducia può essere rilevata in presenza di segnali specifici come la reticenza nella collaborazione o nell'assunzione di responsabilità condivise. In aggiunta, alcuni membri potrebbero dimostrarsi poco attivi nelle discussioni di gruppo o nelle attività cooperative. 
+  - *Identificazione*: la mancanza di fiducia può essere rilevata in presenza di segnali specifici come la reticenza nella collaborazione o nell'assunzione di responsabilità condivise. In aggiunta, alcuni membri potrebbero dimostrarsi poco attivi nelle discussioni di gruppo o nelle attività cooperative. 
   - *Valutazione*: Pericolosità: media; Occorrenza: bassa.
   - *Prevenzione*: 
     - Promuovere incontri periodici: i membri del team hanno la possibilità di esprimere dubbi e preocupazioni, avere una discussione aperta su aspetti rilevanti del progetto e ricevere feedback sul lavoro fatto. In questo contesto va precisato che la costruzione del feedback assume un ruolo fondamentale, in quanto deve essere formulato in maniera tale da risultare costruttivo e presentato come un opportunità di miglioramento anziché una critica;
     - Promuovere la comprensione reciproca: il team si deve sforzare nel comprendere le condizioni e i punti di vista degli altri componenti per creare un clima di progetto positivo indirizzato verso il miglioramento.
 
+=== *RC5 - Context Switch*
+
+Per #glossary("Context Switch") ci si riferisce al cambiamento dei ruoli all'interno del team. Questo passaggio da un ruolo ad un altro comporta la necessità di adattarsi rapidamente alle nuove responsabilità e di comprendere appieno ciò che è stato svolto dal membro del team precedentemente.
+  - *Identificazione*: Difficoltà nel capire cosa è stato fatto precedentmente. 
+  - *Valutazione*: Pericolosità: bassa; Occorenza: alta.
+  - *Prevenzione*:
+      - La persona che ha precedentemente occupato quel ruolo specifico, offre supporto al membro del team che ha assunto ora quella posizione, facilitandone la transizione;
+      - Assistere il responsabile o l'amministratore nella creazione delle attività per il prossimo sprint;
+      - Dedicare il primo giorno  unicamente allo studio di quanto fatto precedentemente per riuscire a sincronizzarsi con il contesto del ruolo.
+
```

Comment:
- nel secondo punto, Responsabile e Amministratore
- manca la menzione del fatto che usiamo il sistema di revisione/verifica per arginare il rallentamento causato dal context switch/cold start, della serie le prime modifiche apportate dal componente che occupa il nuovo ruolo vengono revisionate dal componente che lo occupava in precedenza